src/NMS.AMQP/Provider/Amqp/Message/AmqpNmsMessageFacade.cs

```diff
@@ -271,13 +271,13 @@ public DateTime DeliveryTime
                     case uint _:
                         return new DateTime(621355968000000000L + (long) deliveryTime * 10000L, DateTimeKind.Utc);
                     default:
-                        return syntheticDeliveryTime;
+                        return DateTime.UtcNow; // syntheticDeliveryTime;
```

Comment:
can you revert this, it needs to be the time the message was received by the client Not the time someone asks for the value.

Reply:
ok, about incoming message
in context of sending out, please check on your own, cause to me that syntheticDeliveryTime seemed its not actually used

Reply:
Its not used on send its used on receive if the original message didnt have one set

Reply:
I covered this in my comment about the test case too

Reply:
You can see qpid with the same here:
https://github.com/apache/qpid-jms/blob/198e4f2c7361ef4d62cc173d5f91ce24fea8e80b/qpid-jms-client/src/main/java/org/apache/qpid/jms/provider/amqp/message/AmqpJmsMessageFacade.java

Reply:
And here's a part of one test case thats checking this behaviour on qpid. 

https://github.com/apache/qpid-jms/blob/198e4f2c7361ef4d62cc173d5f91ce24fea8e80b/qpid-jms-client/src/test/java/org/apache/qpid/jms/integration/MessageIntegrationTest.java#L2308

I sent you more links on my comment around testing